modules/rte_mis_school/js/entry_class.js

```diff
@@ -0,0 +1,47 @@
+(function ($, Drupal, once) {
+  Drupal.behaviors.entryClass = {
+    attach: function (context, settings) {
+      $(once('entry-class', 'body', context)).each(function () {
+
+        $('#entry-class-wrapper .entry-class-select').each(function() {
+          // Check if the current value is "_none"
+          if ($(this).val() === '_none') {
+            $(this).children('option[value!="_none"]').remove();
+            // Populate select list with values of selected radio buttons
+            var $wrapper = $('#entry-class-wrapper');
+            var $radioButtons = $('#default-entry-class input[type="radio"]:checked, #optional-entry-class input[type="radio"]:checked');
```

Comment:
Why we are adding `$` with variable name?

Reply:
Updated